controllers/v1/observationsController.js

```diff
@@ -2078,6 +2079,7 @@ module.exports = class Observations extends Abstract {
 
         return resolve(observations);
       } catch (error) {
+        console.log(error)
```

Comment:
@borkarsaish65 if not used please remove the console

Reply:
done